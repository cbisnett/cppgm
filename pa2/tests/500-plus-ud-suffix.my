invalid 1_e+
invalid 1_e+1
invalid 1.0_e+1
eof
