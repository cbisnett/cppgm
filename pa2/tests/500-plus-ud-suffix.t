1_e+
1_e+1
1.0_e+1

