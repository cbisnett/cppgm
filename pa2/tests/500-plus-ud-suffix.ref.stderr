ERROR: invalid character in ud-suffix: 1_e+
ERROR: invalid character in ud-suffix: 1_e+1
ERROR: invalid character in ud-suffix: 1.0_e+1
